restli-server/src/main/java/com/linkedin/restli/server/BaseRestLiServer.java

```diff
@@ -172,8 +175,8 @@ protected RoutingResult getRoutingResult(Request request, RequestContext request
     try
     {
       ServerResourceContext context = new ResourceContextImpl(new PathKeysImpl(), request, requestContext);
-      RestUtils.validateRequestHeadersAndUpdateResourceContext(
-          request.getHeaders(), _customContentTypes, context, requestContext);
+      RestUtils.validateRequestHeadersAndUpdateResourceContext(request.getHeaders(), _supportedAcceptTypes,
+              _customContentTypes, context, requestContext);
```

Comment:
can you fix the tab/spce issue? The diff file shows you might be using inconsistent tab/spaces here.